sdks/python/apache_beam/runners/portability/fn_api_runner_test.py

```diff
@@ -1603,23 +1603,6 @@ def restriction_size(self, element, restriction):
     return restriction.size()
 
 
-class FnApiRunnerSplitTestWithMultiWorkers(FnApiRunnerSplitTest):
```

Comment:
Is there any loss of coverage in removing these tests? I can see how perhaps they're too strict.